template.yaml

```diff
@@ -261,6 +261,50 @@ Resources:
           ScopeName: DeathNotification
       UserPoolId: !Ref UserPool
 
+  DwpIntegrationManagedPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Condition: IsIntegration
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: sqsAccess
+            Effect: Allow
+            Resource:
+              - !Ref DwpDevDeathDeliveryQueue
+              - !Ref DwpTestDeathDeliveryQueue
+              - !Ref DwpStageDeathDeliveryQueue
+            Action:
+              - SQS:GetQueueUrl
+              - SQS:GetQueueAttributes
+              - SQS:ChangeMessageVisibility
+              - SQS:ReceiveMessage
+              - SQS:DeleteMessage
+          - Sid: keyAccess
+            Effect: Allow
+            Action:
+              - kms:GenerateDataKey
+              - kms:Decrypt
+            Resource:
+              - !Ref DwpDevDeathKmsKey
+              - !Ref DwpTestDeathKmsKey
+              - !Ref DwpStageDeathKmsKey
+
+  DwpIntegrationUserGroup:
+    Type: AWS::IAM::Group
+    Condition: IsIntegration
+    Properties:
+      GroupName: IamUserGroup
+      ManagedPolicyArns:
+        - !Ref DwpIntegrationManagedPolicy
+
+  DwpIntegrationUser:
+    Type: AWS::IAM::User
+    Condition: IsIntegration
+    Properties:
+      Groups:
+        - !Ref DwpIntegrationUserGroup
```

Comment:
I think we should either explicitly name this user, or give it a tag which makes it clear from the console what the user is for